src/autocomplete.js

```diff
@@ -274,16 +291,16 @@ class Autocomplete extends Component {
       () => {
         // call the provided callback if it's a callback
         cbToCb(cb)()
-        // if the selectedValue changed
```

Comment:
I want to have a discussion about what we can remove in another issue before we release 1.0.0 :smile: